### ISSUE TITLE
ci: configure a github workflow to deploy coverage reports to cloudflare pages

### ISSUE THREAD

**Issue by carlosdevpereira** [knok-healthcare/figma-assets-to-github-plugin#8]
## Description

The test framework allows to generate code coverage reports with detailed information of what is not covered by tests. This report can be useful for developers and so it would be a nice to have if these reports where made available automatically instead of having to be generated manually.

## Acceptance Criteria

- Create a Cloudflare pages app for the Figma plugin
- Update the "Run unit tests" workflow to have a step running after the coverage report is generated, that deploys the coverage report to the Figma plugin Cloudflare Pages environment (contextualised with the branch that triggered the push event).
- Update README.md with the link to the coverage report in Cloudflare

---

Depends on: https://github.com/knok-healthcare/figma-assets-to-github-plugin/issues/6
Depends on: #7
